OperatingSystems.md

```diff
@@ -158,6 +158,23 @@ Type|Macro
 ---|---
 Identification|`__ECOS`
 
+## [Emscripten](https://en.wikipedia.org/wiki/Emscripten) ##
+
+Type|Macro|Format|Description|Availability
+---|---|---|---|---
+Identification|`EMSCRIPTEN`| | |Defined as of 1.0.1a, gone as of 1.30.6
+Identification|`__EMSCRIPTEN__`| | |Defined as of 1.3.7
+Version|`__EMSCRIPTEN_major__`|V|V = Major version|Defined as of 1.21.4
+Version|`__EMSCRIPTEN_minor__`|R|R = Minor version|Defined as of 1.21.4
+Version|`__EMSCRIPTEN_tiny__`|P|P = Patch level|Defined as of 1.21.4
```

Comment:
Those last 3 are not actually builtin but are defined in `emscripten/version.h` (this is also included via `emscirpten.h`) these days.. just FYI.

Reply:
That's worth noting here, thanks!